model/clusters_mgmt/v1/external_auth_config_type.model

```diff
@@ -14,15 +14,38 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// ExternalAuthConfig configuration
-struct ExternalAuthConfig {
-    // Boolean flag indicating if the cluster should use an external authentication configuration.
+// Represents an external authentication configuration
+class ExternalAuthConfig {
+    // Boolean flag indicating if the cluster should use an external authentication configuration for ROSA HCP clusters.
     //
     // By default this is false.
     //
     // To enable it the cluster needs to be ROSA HCP cluster and the organization of the user needs
     // to have the `external-authentication` feature toggle enabled.
+    //
+    // For ARO HCP clusters, please use the "State" property to enable/disable this feature instead.
```

Comment:
nitpick: I would avoid the word "please" here, as it's not a polite request - it just won't work to use the boolean, so a direct tone is more appropriate: `For ARO HCP clusters, use the "State" property [...]`.